modin/experimental/core/execution/native/implementations/hdk_on_native/test/test_dataframe.py

```diff
@@ -2062,19 +2064,21 @@ def test_cat_codes(self):
 
 
 class TestSort:
+    # In order for the row order to be deterministic after sorting,
```

Comment:
So, we just removed the previous test, right? We should just have skipped it or marked as xfailed.

Reply:
The test is not removed, but modified. It was not quite correct, because HDK does not guarantee the rows order after sorting. For example, when sorting the following frame:
```
A   B
1   1
2   3
1   2
```
 by the column A we can get the following 2 results and both are valid:
```
A   B
1   1
1   2
2   3
```
```
A   B
1   2
1   1
2   3
```

Reply:
Yes, we have in the implementation notes to the supported API.

Reply:
Good, thanks!